python/robot_fingers/ros/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""ROS-related classes/functions."""
+import rclpy
+import rclpy.node
+import rclpy.qos
+from std_msgs.msg import String
+from std_srvs.srv import Empty
+
+
+class NotificationNode(rclpy.node.Node):
+    """Simple ROS node for communication with other processes."""
+
+    def __init__(self, name):
+        super().__init__(name)
+
+        # quality of service profile keep the last published message for late
+        # subscribers (like "latched" in ROS 1).
+        qos_profile = rclpy.qos.QoSProfile(
+            depth=1,
+            durability=rclpy.qos.DurabilityPolicy.TRANSIENT_LOCAL,
+            history=rclpy.qos.HistoryPolicy.KEEP_LAST,
+            reliability=rclpy.qos.ReliabilityPolicy.RELIABLE,
+        )
+
+        self._status_publisher = self.create_publisher(
+            String, "~/status", qos_profile
```

Comment:
I would suggest to add a prefix (e.g. trifinger/status), and to set the string as global const string defined at the beginning of the file

Reply:
The prefix is implicitly given through the `~`.  The resulting topic name will be `/node_name/status`.